astro/src/content/blog/2023-hacktoberfest.mdx

```diff
@@ -0,0 +1,52 @@
+
+To see all of FusionAuth's issues in one place see the [full issue search](https://github.com/search?q=user%3AFusionAuth+label%3Ahacktoberfest+state%3Aopen&type=Issues&ref=advsearch&l=&l=).
+
+This year for the Hacktoberfest we choose to focus on all of our JavaScript repositories. I have added the tag `hacktoberfest` to each one of the following repositories and even added an issue for simply updating the packages to their latest version. 
+
+
+* [Angular ](https://github.com/fusionauth/fusionauth-quickstart-javascript-angular-web)- The Authorization Code grant using the Angular framework
+* [Consents ](https://github.com/FusionAuth/fusionauth-example-express-consents)- Example using advanced1 registration forms and consents
+* [Deeplinking ](https://github.com/FusionAuth/fusionauth-example-node-deeplink)- Example returning users to the same page they logged in on
+* [Device grant ](https://github.com/fusionauth/fusionauth-example-device-grant)- An example of the Device Authorization grant
+* [Express API ](https://github.com/FusionAuth/fusionauth-example-express-api)- Express api which uses the Cookie Access Token
+* [Family management ](https://github.com/fusionauth/fusionauth-example-family)- Family management and consent creation
+* [FusionAuth SSO ](https://github.com/FusionAuth/fusionauth-example-node-sso)- Example of SSO between two different custom applications
+* [Gaming and device grant ](https://github.com/FusionAuth/fusionauth-example-gaming-device-grant)- Example using the Device Authorization grant to provide authentication to a game.
+* [Gatsby OAuth ](https://github.com/fusionauth/fusionauth-example-gatsby)- An example of using Gatsby with the Authorization Code grant and PKCE
+* [JWT Auth and a Microservices gateway ](https://github.com/FusionAuth/fusionauth-example-node-services-gateway-jwtauth)- API gateway and microservices secured using JWT auth
+* [Javascript JWT ](https://github.com/fusionauth/fusionauth-example-javascript-jwt)- JWT creation and decoding examples with javascript
+* [Magic links login ](https://github.com/FusionAuth/fusionauth-example-nextjs-magic-links)- nextjs app which uses magic links for authentication
+* [Microservices gateway ](https://github.com/FusionAuth/fusionauth-example-node-services-gateway)- API gateway and microservices
+* [Multi-tenant application ](https://github.com/FusionAuth/fusionauth-example-node-multi-tenant)- Two nodejs applications in different tenants, living in different domains.
+* [Next.js Single Sign-On ](https://github.com/FusionAuth/fusionauth-example-nextjs-single-sign-on)- Single sign-on with Next.js and FusionAuth
+* [Node OAuth ](https://github.com/fusionauth/fusionauth-example-node)- Login with the Authorization Code grant
+* [React Native ](https://github.com/fusionauth/fusionauth-example-react-native-0-71)- The Authorization Code grant for a React Native mobile application
```

Comment:
Same worries about archiving here.

Reply:
same thought, ill cross that bridge. I think it will still count for them.